spec/main.tex

```diff
@@ -981,24 +995,28 @@ \subsubsection{Threshold (i.e., $t$-out-of-$n$) Key Generation}
         \item Upon receiving $h_j$ from all  parties: abort if 
         $h_i \neq h_j$ for some $j \in [n]$.
     \end{itemize}
-      \item Send $(\rid_i, \vec{S}_i, A_i, u_i)$ to all parties.
+      \item Send $(\rid_i, \vec{S}_i, A_i, u_i, c_i)$ to all parties.
       \item For all $j \neq i$, send $\sigma_{i,j}$ to $P_j$ via private channel.
     \end{itemize}
 
   \item[\textbf{Round 3.}] 
-Upon receiving $(\rid_j, \vec{S}_j, A_j, u_j)$ and $\sigma_{j,i}$ from all parties:
+    Upon receiving $(\rid_j, \vec{S}_j, A_j, u_j, c_j)$ and $\sigma_{j,i}$ from all parties:
     \begin{itemize}
       \item For each party $j \neq i$:
         \begin{itemize}
         \item Check that $\vec{S}_j$ has length $t$.
-          \item Assert $V_j = H(\sid\| n\| j\| t\| \rid_j\| \vec{S}_j\| A_j\| u_j)$.
+        \item Assert $V_j = H(\sid\| n\| j\| t\| \rid_j\| \vec{S}_j\| A_j\| u_j \| c_j)$.
         \item Define 
-        $F_j(x) = \sum_{k \in [t]} x^k \cdot S_{j,k}$.
-          \item Assert $\sigma_{j,i} \cdot G  = F_j(i + 1)$.
+          $F_j(x) = \sum_{k \in [t]} x^k \cdot S_{j,k}$.
+        \item Assert $\sigma_{j,i} \cdot G  = F_j(i + 1)$.
```

Comment:
Should be indented one more level inside `itemize`

Reply:
The whole document formatting is messed up, there are many places where `itemize` and `item` have the same intent level

Reply:
Yeah, but would be better if we didn't introduce new ones 🙃

Reply:
I'd suggest to move all stylistic suggestions to when we have a formatter

Reply:
Sure. I'll resolve all those comments for now